### PR TITLE
Update local CDA setup documentation

### PR DESCRIPTION
**Pull request by mabely**
#### What
Update local CDA setup documentation

#### Ticket

[board ticket description](https://dsdmoj.atlassian.net/browse/AAC-91)

#### Why
Some Postgres setup errors were encountered when setting up CDA locally according to the README.

#### How
To avoid possible Postgres errors encountered when trying to run `rails console` to generate OAuth2 credentials, the readme now sets the CDA database URL and creates the database first which will hopefully prevent this for others.

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -40,7 +41,20 @@ git clone git@github.com:ministryofjustice/laa-court-data-adaptor.git
...
+DATABASE_URL=postgres://localhost/laa_court_data_adaptor_development
+INLINE_SIDEKIQ=true
+```
+
+- setup CDA database
+```
+rails db:create db:migrate db:seed
+```
+
+- generate OAuth2 `client_credentials` - for the UI (see end of section for postgres error)
```
Comment (kmahern) [inline docs/development.md:57]: @mabely not sure what is meant by see end of section for postgres error?

Comment (mabely) [reply docs/development.md]: @kmahern I've removed that, must have forgotten to remove by accident. I also added an alternative to setting up the db in the same commit so it aligns with CDA documentation, what I added didn't work for me but just so it doesn't diverge I included it - just FYI.

File: docs/development.md [hunk 2]
```diff
@@ -21,7 +21,8 @@ cd .../hmcts-common-platform-mock-api
 - create dummy data
 ```
 # create some data
-$ rails console
+rails mock:demodata:load
```
Comment (kmahern) [inline docs/development.md:24]: @mabely I think if we document to run rails` mock:demodata:load` we do not need ```
rails console
Pry> FactoryBot.create(:realistic_prosecution_case, defendants: FactoryBot.create_list(:realistic_defendant, 3))
``` (as I think these would do the same thing).

**Comment by kmahern** [comment]
One small comment otherwise lgtm 👍

**Review by kmahern — COMMENTED** [review]
👍

**Review by kmahern — APPROVED** [review]

File: docs/development.md (hunk 2 above)
Comment (mabely) [reply docs/development.md:24]: Removed it - thanks for spotting it!

**Pull request merged**